ledger/ledger-api-tests/suites/src/main/scala/com/daml/ledger/api/testtool/suites/v1_8/TransactionServiceVisibilityIT.scala

```diff
@@ -89,14 +90,26 @@ class TransactionServiceVisibilityIT extends LedgerTestSuite {
         dkkTree <- eventually("transactionTreeById3") {
           delta.transactionTreeById(tree.transactionId, dkk_bank)
         }
-
+        aliceTrees <- alpha.transactionTrees(alice)
```

Comment:
Since transactionTreeById boils down to different API usage than transactionTrees: from now on we test both
(concrete motivation: In memory fan out is short circuited for point-wise lookups)